src/reference/asciidoc/testing.adoc

```diff
@@ -150,6 +150,19 @@ public class KafkaStreamsTests {
 }
 ----
 
+'''
+
+The `brokerProperties` attribute of `@EmbeddedKafka` supports standard Spring property placeholder resolution from the `Environment`.  For example:
+[source, java]
+----
+@TestPropertySource(locations = { "classpath:/kafka-broker.properties" })
+@EmbeddedKafka(topics = { "any-topic" },
+		brokerProperties = { "log.dir=${kafka.broker.logs-dir}", "listeners=PLAINTEXT://localhost:${kafka.broker.port}", "auto.create.topics.enable=true" })
+----
+In this example, the property placeholders `${kafka.broker.logs-dir}` and `${kafka.broker.port}` are resolved from the Spring `Environment`.   The `Environment` contains system properties, the system environment, _and_ properties from the classpath resource `/kafka-broker.properties`.  All of these property sources are consulted when resolving property placeholders.  If the property placeholder is resolved by the `Environment`, the resolved value will be supplied to the broker, otherwise the original value will be supplied.
+
+*_N.B._*: Property placeholders are _not_ supported for other attributes of `@EmbeddedKafka` at this time.
```

Comment:
We don't need to tell this.
The `EmbeddedKafka.brokerProperties()` JavaDoc has to mention that it is aware about PP resolution.